docs/config.md

```diff
@@ -17,48 +17,46 @@ This is tested for the Duo Security API version V2.
+   ![alt text](images/duodoc1.png)
+3. In the **Protect an Application**  page, select **Web SDK** from the list. **Web SDK** credentials are **mandatory**.
 
-    ![alt text](images/duodoc4.png)
-4. Once the integration is created, you are given a **Secret key** and an **Integration key** for your integration. You can use these along with your Duo host when accessing Duo Security APIs.
+   ![alt text](images/duodoc2.png)
+4. Once the integration is created, you are given a **Client ID** and a **Client Secret** for your integration. You can use these along with your Duo API hostname when accessing Duo Security APIs.
 
-    ![alt text](images/auth.png)
+   ![alt text](images/duodoc3.png)
 5.  You can also configure the **Admin API** credentials if you need to validate the mobile numbers. When you verify the mobile number, use only these credentials. Navigate back to the **Protect an Application**  page and select  **Admin API**  from the list.  Once the Integration is created, you are given a **Secret key** and an **Integration key** for your integration.
 
+    ![alt text](images/duodoc4.png)
     ![alt text](images/duodoc5.png)
  >> NOTE : If you can not see the type “Admin API” in the dropdown, contact the Duo team through support@duosecurity.com and ask for Admin API permission.
 
 6. When configuring the Admin API, select the Grant read resource permission.
 
- ![alt text](images/duodoc5.png)
+   ![alt text](images/duodoc6.png)
 
  >> NOTE : This step is mandatory if you need to verify the user's mobile number in the user store with the mobile number in Duo Security. This is configured in step 4 of [Deploying Duo Security artifacts](#deploying-duo-security-artifacts).
 
 ### Deploying Duo Security artifacts
 To download the authenticator and artifacts, go to the [WSO2 store](https://store.wso2.com/store/assets/isconnector/list?q=%22_default%22%3A%22duo%22).
 
  1. Place the duoauthenticationendpoint.war file into the <IS_HOME>/repository/deployment/server/webapps directory.
-     
- 2. Place the org.wso2.carbon.identity.authenticator.duo-1.x.x.jar file into the <IS_HOME>/repository/components/dropins directory.
-    >> NOTE : If you want to upgrade the Duo Authenticator in your existing IS pack, please refer [upgrade instructions](https://docs.wso2.com/display/ISCONNECTORS/Upgrading+an+Authenticator).
-
- 3. Place the [okio-1.9.0.jar](hhttps://search.maven.org/remote_content?g=com.squareup.okio&a=okio&v=1.9.0) into the <IS_HOME>/repository/components/lib directory.
 
-    >> NOTE : You may have done this step already if you configured the [Duo Security Provisioning Connector](https://docs.wso2.com/display/ISCONNECTORS/Configuring+Duo+Security+Provisioning+Connector). If so, you can skip this step.
+ 2. Place the org.wso2.carbon.identity.authenticator.duo-2.x.x.jar file into the <IS_HOME>/repository/components/dropins directory.
```

Comment:
Doesn't this need to be changed to 3.x.x?

Reply:
Fixed with b01b0424234c927451a770a873a358d5e71068bc